plugins/@grouparoo/mixpanel/src/lib/export/exportProfile.ts

```diff
@@ -0,0 +1,131 @@
+import {
+  ExportProfilePluginMethod,
+  Errors,
+  DestinationSyncOperations,
+} from "@grouparoo/core";
+import Mixpanel from "../client/mixpanel";
+import { connect } from "../connect";
+
+const deleteProfileOrClearGroups = async (
+  client: Mixpanel,
+  syncOperations: DestinationSyncOperations,
+  profile: any
+) => {
+  if (syncOperations.delete) {
+    await client.ingestion.profile.delete(profile["$distinct_id"]);
+  } else {
+    if (syncOperations.update) {
+      // clear groups
+      await client.ingestion.profile.update(profile["$distinct_id"], {
+        groups: [],
+      });
+    }
+  }
+};
+
+export const exportProfile: ExportProfilePluginMethod = async ({
```

Comment:
It's nice to not have an API rate limit, and that we can do the "simpler" thing of just worrying about 1 profile at a time.